app/app.pro

```diff
@@ -6,18 +6,11 @@
 
 ! include( ../util/common.pri ) { error( Could not find the common.pri file! ) }
 
-QT += core widgets gui xml multimedia svg network
-
 TEMPLATE = app
-TARGET = pencil2d
-QMAKE_APPLICATION_BUNDLE_NAME = Pencil2D
```

Comment:
I assume this is no longer necessary, because we have the info.plist defined?
Qt doesn't say much about that specific environment variable...